php_memcached.c

```diff
@@ -86,7 +86,7 @@ static int php_memc_list_entry(void) {
 /****************************************
   Payload value flags
 ****************************************/
-#define MEMC_CREATE_MASK(start, n_bits) (((1 << n_bits) - 1) << start)
+#define MEMC_CREATE_MASK(start, n_bits) (((1U << n_bits) - 1) << start)
```

Comment:
I did a bit of reading, found a note that Coverity might interpret `1U` as u_char rather than int: https://community.synopsys.com/s/article/False-positive-for-coverity-sees-1-and-1U-as-8-bit-type-variables

The fix would be to use `1UL` to make it explicitly 32-bit (unsigned long)

Reply:
but 1UL is 64-bit

Reply:
Right, there's a part of my brain that I keep on a [shelf](https://en.wikipedia.org/wiki/64-bit_computing#64-bit_data_models) for this